onboard/rocks_catalog_actions_test.go

```diff
@@ -0,0 +1,125 @@
+package onboard_test
+
+import (
+	"context"
+	"errors"
+	"testing"
+
+	"github.com/treeverse/lakefs/catalog"
+
+	"github.com/treeverse/lakefs/graveler"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/treeverse/lakefs/block"
+	"github.com/treeverse/lakefs/onboard"
+
+	"github.com/treeverse/lakefs/logging"
+
+	"github.com/golang/mock/gomock"
+	"github.com/treeverse/lakefs/onboard/mock"
+)
+
+const (
+	repoID      = graveler.RepositoryID("some-repo-id")
+	metaRangeID = graveler.MetaRangeID("some-mr-id")
+	commitID    = graveler.CommitID("some-commit-id")
+	committer   = "john-doe"
+	msg         = "awesome-import-commit"
+)
+
+var (
+	innerIt = &mockInventoryIterator{
+		rows: []block.InventoryObject{
+			{
+				Bucket:          "bucket-1",
+				Key:             "key-1",
+				Size:            1024,
+				Checksum:        "checksum-1",
+				PhysicalAddress: "/some/path/to/object",
+			},
+		}}
+	validIt = onboard.NewInventoryIterator(innerIt)
+)
```

Comment:
move into the test function or create a function that returns this value.
don't use global variable with state.

Reply:
Done.